cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/service/impl/CoordinateCleaningManagerImpl.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.diging.cord19.explorer.core.service.impl;
+
+import java.time.OffsetDateTime;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+
+import edu.asu.diging.cord19.explorer.core.data.TaskRepository;
+import edu.asu.diging.cord19.explorer.core.model.task.TaskType;
+import edu.asu.diging.cord19.explorer.core.model.task.impl.TaskImpl;
+import edu.asu.diging.cord19.explorer.core.service.CoordinateCleaningManager;
+import edu.asu.diging.cord19.explorer.core.service.worker.CoordinateCleaner;
+;
+
+@Component
+public class CoordinateCleaningManagerImpl implements CoordinateCleaningManager {
+    
+    @Autowired
+	private TaskRepository taskRepo;
+	
+	@Autowired
+    private CoordinateCleaner cleaner;
```

Comment:
indentation

Reply:
I am not sure what to do about this. All the indentation matches on mine. I reformatted just in case.